### ISSUE TITLE
TypeError: Cannot read property 'channels' of undefined

### ISSUE THREAD

**Issue by minemidnight** [abalabahaha/eris#293]
```
TypeError: Cannot read property 'channels' of undefined
7|oxyl     |     at Shard.wsEvent (/home/dbots/oxyl/node_modules/eris/lib/gateway/Shard.js:890:46)
7|oxyl     |     at WebSocket.ws.onmessage (/home/dbots/oxyl/node_modules/eris/lib/gateway/Shard.js:1522:34)
7|oxyl     |     at WebSocket.onMessage (/home/dbots/oxyl/node_modules/ws/lib/EventTarget.js:99:16)
7|oxyl     |     at emitOne (events.js:115:13)
7|oxyl     |     at WebSocket.emit (events.js:210:7)
7|oxyl     |     at Receiver._receiver.onmessage (/home/dbots/oxyl/node_modules/ws/lib/WebSocket.js:143:47)
7|oxyl     |     at Receiver.dataMessage (/home/dbots/oxyl/node_modules/ws/lib/Receiver.js:389:14)
7|oxyl     |     at Receiver.getData (/home/dbots/oxyl/node_modules/ws/lib/Receiver.js:330:12)
7|oxyl     |     at Receiver.startLoop (/home/dbots/oxyl/node_modules/ws/lib/Receiver.js:165:16)
7|oxyl     |     at Receiver.add (/home/dbots/oxyl/node_modules/ws/lib/Receiver.js:139:10)
7|oxyl     |     at TLSSocket._ultron.on (/home/dbots/oxyl/node_modules/ws/lib/WebSocket.js:139:22)
7|oxyl     |     at emitOne (events.js:115:13)
7|oxyl     |     at TLSSocket.emit (events.js:210:7)
7|oxyl     |     at addChunk (_stream_readable.js:266:12)
7|oxyl     |     at readableAddChunk (_stream_readable.js:253:11)
7|oxyl     |     at TLSSocket.Readable.push (_stream_readable.js:211:10)
7|oxyl     |     at TLSWrap.onread (net.js:585:20)
```

Getting this error since updating to dev for the fix for channel type 4

**Comment by aurieh** [comment]
While I mentioned this in #js_eris, `guild` in that context is undefined. Fixing this should be as easy as `var guild = this.client.guilds.get(this.client.channelGuildMap[packet.d.id])`.

**Comment by TTtie** [comment]
can reproduce, fix pr is at #294 (possibly potato code)

**Comment by minemidnight** [comment]
@aurieh yeah, i've fixed it in my own code just by using `nano` for now but i was too lazy to make a pr, so i decided to inform instead
